src/UrlParams.ts

```diff
@@ -229,6 +243,8 @@ export const getUrlParams = (
     perParticipantE2EE: parser.getFlagParam("perParticipantE2EE"),
     skipLobby: parser.getFlagParam("skipLobby"),
     returnToLobby: parser.getFlagParam("returnToLobby"),
+    viaServers: parser.getParam("viaServers"),
+    customHomeserver: parser.getParam("customHomeserver"),
```

Comment:
```suggestion
    homeserver: parser.getParam("homeserver"),
```

What do you think about this naming?

Reply:
I was also considering this.
I wanted to make sure its clear that this is not required. But I guess since its called `defaultHomeserver` in the rest of the code `homeserver` should be sufficient.